config/routes.rb

```diff
@@ -93,7 +93,6 @@
 
   get "/course/svn" => "site#svn"
   get "/sfc" => "site#sfc"
-  get "/trademark" => "site#trademark"
```

Comment:
another nice catch! :)

Only a minor improvement: when you have a web application that has been online, you should avoid break previous links. There may exist links in some other website for this URL, so we need to find a solution. If the content was completely removed, one solution is redirecting to the homepage. Since our content was just moved to somewhere else, we should redirect for the new location:

  `get "/trademark" => redirect('/about/trademark')`